clash-prelude/src/Clash/Sized/Fixed.hs

```diff
@@ -156,6 +157,8 @@ import Clash.XException
 >>> let n = $$(fLit pi) :: SFixed 4 4
 -}
 
+type role Fixed representational nominal nominal
```

Comment:
If this role annotation corresponds to the inferred rule, could you remove it please.